CHANGELOG.md

```diff
@@ -1,3 +1,75 @@
+## 9.2.2 "graphite-graviton" (2020-04-30)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| fix |  **testing/testbed:** emit pointer events for test element click ([d7b1bbe](https://github.com/angular/components/commit/d7b1bbe)) |
```

Comment:
Out of curiosity: Do you know why this is "fix" and not "bug fix" as for the others?

Reply:
Because I did it manually and wrote the wrong thing (it was `feat` before when I considered it a fix). Pushed a correction.